tests/unit/IdGeneratorTest.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+declare(strict_types=1);
+
+use PHPUnit\Framework\TestCase;
+use Instrumentation\Aws\Xray\AwsXrayIdGenerator;
+use OpenTelemetry\Sdk\Trace\SpanContext;
+
+class AwsXrayIdGeneratorTest extends TestCase
+{
+    /**
+     * @test
+     */
+    public function GeneratedTraceIdIsValid()
+    {
+        $idGenerator = new AwsXrayIdGenerator();
+        $traceId = $idGenerator->generateTraceId();
+
+        $this->assertEquals(1, preg_match(SpanContext::VALID_TRACE, $traceId));
+    }
+    
+    /**
+     * @test
+     */
+    public function GeneratedTraceIdIsUnique()
+    {
+        $idGenerator = new AwsXrayIdGenerator();
+        $traceId1 = $idGenerator->generateTraceId();
+        $traceId2 = $idGenerator->generateTraceId();
+
+        $this->assertFalse($traceId2 == $traceId1);
```

Comment:
```suggestion
        $this->assertFalse($traceId2 === $traceId1);
```

[PHP is silly](http://phpsadness.com/sad/52) and [thinks a lot of things are equivalent if you use `==`](http://phpsadness.com/sad/47).

Reply:
Fixed